cmd/virt-launcher/virt-launcher.go

```diff
@@ -206,6 +208,7 @@ func main() {
 	watchdogInterval := flag.Duration("watchdog-update-interval", defaultWatchdogInterval, "Interval at which watchdog file should be updated")
 	readinessFile := flag.String("readiness-file", "/tmp/health", "Pod looks for tihs file to determine when virt-launcher is initialized")
 	gracePeriodSeconds := flag.Int("grace-period-seconds", 30, "Grace period to observe before sending SIGTERM to vm process.")
+	allowEmulation := flag.Bool("allow-emulation", false, "If set to true then software emulation will be used if /dev/kvm is not available.")
```

Comment:
Can we in addition support an ENV var for this?

Then we can easily combine it with PodPresets, i.e. if you want emulation then you setup a PodPreset to match all launchers setting allow-emulation as an env var.

Reply:
You can always map an ENV on the Pod to a commandline flag in the "command" section. How can we add this with PodPresets? You can't set an ENV var through a pod preset.

Why would I want to run parts of my cluster emulated?

Reply:
Ok now I see how it would work with a PodPreset

Reply:
Still, a commandline argument can always be mapped via an ENV variable.

Reply:
Will add it.